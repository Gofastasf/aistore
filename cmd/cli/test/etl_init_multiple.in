// RUN k8s
# Test multiple ETL initialization in a single spec file
ais etl rm --all // IGNORE
echo -e "name: test-etl1\nruntime:\n    image: aistorage/transformer_hello_world:latest\n---\nname: test-etl2\nruntime:\n    image: aistorage/transformer_hello_world:latest" > etl-config.yaml // IGNORE
ais etl init -f etl-config.yaml --name test-etl1 2>&1 || true
ais etl init -f etl-config.yaml // IGNORE
ais etl show | grep Running | wc -l
rm -f etl-config.yaml // IGNORE