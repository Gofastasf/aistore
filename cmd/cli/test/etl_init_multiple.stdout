Error: cannot use override flags [--name] with a multi-ETL file; remove these flags or split into separate files
2